plugins/infobright-bulk-loader/impl/src/main/java/org/pentaho/di/trans/steps/infobrightoutput/InfobrightLoaderMeta.java

```diff
@@ -55,7 +55,7 @@
         description = "BaseStep.TypeTooltipDesc.InfobrightOutput",
         categoryDescription = "i18n:org.pentaho.di.trans.step:BaseStep.Category.Bulk",
         image = "BLKIB.svg",
-        documentationUrl = "http://wiki.pentaho.com/display/EAI/Infobright+Loader",
+        documentationUrl = "https://pentaho-community.atlassian.net/wiki/spaces/EAI/pages/386800365/Infobright+Loader",
```

Comment:
as suggested by @urena-kenny via slack :https://hitachivantara-eng.slack.com/archives/C03H06W1GR0/p1657739868984469?thread_ts=1657707247.789619&cid=C03H06W1GR0

If you change the URL to https://pentaho-community.atlassian.net/wiki/display/EAI/Infobright%20Loader It will redirect to the latest version

Reply:
Updated as per comments